### ISSUE TITLE
Does not support accessibility features & not have props to provide it

### ISSUE THREAD

**Issue by sandeep8080** [wwan5803/react-modal-resizable-draggable#57]
Modal created through this are not accessible. Moreover, there are no props provided in the library to support accessibility for a Modal.
